### ISSUE TITLE
Docker Hub Sync error

### ISSUE THREAD

**Issue by hotio** [meeDamian/sync-readme#7]
I'm no longer able to sync with docker hub. Problem on my end? Problem with Docker Hub? Or the action?

Returned error is `ERR: Unable to update description on Docker Hub`, so the login still seems to work.

**Comment by meeDamian** [comment]
Has your description changed?  Are there any special characters in it?  Any pointers?

**Comment by hotio** [comment]
No, no changes at all, just tried it again with a README that was working fine add a new line and it failed. Just one day they all failed....

**Comment by hotio** [comment]
I found the issue, I was using a token, which up until till recently worked just fine, it also works great to push images to the registry, however it no longer works to update the description. I switched to my account password and it synced.

**Comment by meeDamian** [comment]
Reopening, as it's a persistent issue…  tl;dr: impossible to interact with it through API token (permission was removed), and impossible to obtain API token using password when 2FA is enabled.

More details: https://github.com/docker/hub-feedback/issues/1927
